Makefile

```diff
@@ -13,6 +13,10 @@ audit:
 build: generate-prod
 	go build -tags 'production' -o $(BINPATH)/dp-frontend-cookie-controller -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
 
+lint:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2
```

Comment:
Is it worth upgrading to latest and go version 1.19?

Reply:
As discussed this will be done consistently.